### PR TITLE
change-account-ledger

### PR DESCRIPTION
**Pull request by sakilbhedaagile**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
- Account selection in ledger popup
- Update ledger transaction based on selected acount

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: src/app/ledger/components/updateLedgerEntryPanel/updateLedgerEntryPanel.component.ts [hunk 1]
```diff
@@ -600,4 +615,9 @@ export class UpdateLedgerEntryPanelComponent implements OnInit, AfterViewInit, O
   public showQuickAccountModal() {
     this.showQuickAccountModalFromUpdateLedger.emit(true);
   }
+
+  public changeBaseAccount(obj) {
+    console.log('changes', obj)
```
Comment (sarfarazansari) [inline src/app/ledger/components/updateLedgerEntryPanel/updateLedgerEntryPanel.component.ts]: don't open consoles

**Comment by walkover101** [comment]
@mustafa-walkover please let me know what to test in this card.

**Comment by mustafa-walkover** [comment]
@Meghnawalkover  nothing to test in this card, this is the card created for raising PR, avoid this card

**Comment by mustafa-walkover** [comment]
not a bug, so closing this card

**Pull request closed without merging**